examples/pubsub/tutorial_pubsub_configuration.c

```diff
@@ -0,0 +1,180 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/* Includes */
+#include <stdio.h>
+#include <signal.h>
+#include <open62541/server.h>
+#include <open62541/server_config_default.h>
+#include <open62541/plugin/pubsub.h>
+
+/* Global variables */
+volatile UA_Boolean g_running = true;
+
+/* Initialization of signal handler */
+void initSigHandler(void);
+
+/* Function to give user information about correct usage */
+void usage_info(void);
+
+/* Add variable nodes */
+void addVariables(UA_Server *server);
+
+/***** MAIN FUNCTION **********************************************************************************/
+int main(int argc, char** argv)
+{
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+    UA_Boolean loadPubSubFromFile = UA_FALSE;
+#endif
+    UA_UInt16 port = 4840;
+
+    /* 1. Check arguments and set name of PubSub configuration file*/
+    switch(argc)
+    {
+        case 2:
+            port = (unsigned short)atoi(argv[1]);
+            break;
+        case 3:
+            port = (unsigned short)atoi(argv[1]);
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+            loadPubSubFromFile = UA_TRUE;
+#endif
+            break;
+
+        default:
+            usage_info(); 
+    }
+
+    /* Initialize Signal Handler for exiting the process */
+    initSigHandler();
+
+    /* 2. Initialize Server */
+    UA_Server *server = UA_Server_new();
+    UA_ServerConfig *config = UA_Server_getConfig(server);
+    UA_ServerConfig_setMinimal(config, port, NULL); /* creates server on default port 4840 */
+
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+    if(loadPubSubFromFile) 
+    {
+        UA_Server_setPubSubConfigFilename(server, argv[2]);
+    }
```

Comment:
In the open62541 codebase are normaly no brackets used for one-line if statements.